R/ReferenceTests/Compat.toml

```diff
@@ -3,14 +3,20 @@ ImageInTerminal = "0.2-0.3"
 Images = "0.6-0.17"
 julia = "0.6-0.7"
 
-["0.0-0.5"]
+["0.0-0.6"]
 ColorTypes = "0.4-0.7"
 FileIO = "0.4-1.0"
 
 ["0.3-0.5"]
 DeepDiffs = "0.1-1.1"
 
 ["0.4-0.5"]
-ImageInTerminal = "0.3"
 Images = "0.15-0.17"
 julia = "0.7-1.3"
+
+["0.4-0.6"]
+ImageInTerminal = "0.3"
+
+["0.6"]
+Images = "0.15-0.18"
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.